Automattic/jetpack
# Add TypeScript build for mu-wpcom

## Proposed changes:

This PR adds the functionality to build TypeScript for mu-wpcom. 
I tried to align with other TS projects in `./projects/packages`. I'd appreciate it if you let me know anything else required to include js files in the plugin 🙏 

This PR is part of https://github.com/Automattic/wp-calypso/issues/82218, as we don't want to add a feature to ETK.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


No
## Testing instructions:






* Run `pnpm jetpack build packages/jetpack-mu-wpcom` (or `pnpm jetpack watch packages/jetpack-mu-wpcom`)
* See if you have built output in `packages/jetpack-mu-wpcom/build`
